app/assets/stylesheets/_state-file.scss

```diff
@@ -298,6 +314,11 @@
   .bulleted ul {
     padding-left: 3rem;
     list-style: disc;
+
+    li {
+      list-style: disc;
+      margin-top: 2rem;
+    }
```

Comment:
i thought the `li` items inherit from the `ul`, do we need to set the `list-style` again?

Also, is `.bulleted` a class that was not used for any elements previously but was defined? can we use `.list--bulleted` instead which seems more widely used?

Reply:
You're right - we didn't need to duplicate the list-style. Also, we did some investigating and learning and found that we could simplify things somewhat, using a combination of existing styles with one override.